lib/query.js

```diff
@@ -11,7 +11,7 @@ const MongooseError = require('./error/mongooseError');
 const ObjectParameterError = require('./error/objectParameter');
 const QueryCursor = require('./cursor/QueryCursor');
 const ReadPreference = require('./driver').get().ReadPreference;
-const applyGlobalMaxTimeMS = require('./helpers/query/applyGlobalMaxTimeMS');
+const { applyGlobalMaxTimeMS, applyGlobalDiskUse } = require('./helpers/query/applyGlobalOption');
```

Comment:
Is Object destructuring ok? 
Unit tests seem to be fine, but what about Browser Support?

Reply:
Should be fine, we're using Webpack which should handle that.

Reply:
Yeah destructuring should be fine here